frontend/src/cards/DisparityBarChartCard.tsx

```diff
@@ -138,7 +138,7 @@ function DisparityBarChartCardWithKey(props: DisparityBarChartCardProps) {
                 Population percentages on this graph add up to over 100% because
                 the racial categories reported for{" "}
                 {metricConfig.fullCardTitleName} include Hispanic as both a
-                racial and ethnicity category. As a result, Hispanic individuals
+                racial and ethnic category. As a result, Hispanic individuals
```

Comment:
from slack: "as both a race and ethnicity category" might align grammatically with rest of copy